.hlxignore

```diff
@@ -2,4 +2,8 @@
 *.md
 LICENSE
 test/*
+build/*
```

Comment:
Do we need to include build dir here?
I don't see any build directory in the root directory.

Reply:
we'll have one soon. 👍 
that's why.